cmd/antrea-agent/options.go

```diff
@@ -306,6 +306,20 @@ func (o *Options) validateAntreaIPAMConfig() error {
 	return nil
 }
 
+func (o *Options) validateMulticlusterConfig(encapMode config.TrafficEncapModeType) error {
+	if (features.DefaultFeatureGate.Enabled(features.Multicluster) || o.config.Multicluster.Enable) &&
+		encapMode != config.TrafficEncapModeEncap {
```

Comment:
encapMode is required only by EnableStretchedNetworkPolicy.

Reply:
AFAIK, Antrea MC gateway also requires encapMode? @luolanzone